README.md

```diff
@@ -1,33 +1,28 @@
-| [intercom__conversation_metrics](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__conversation_metrics.sql)                                 | Each record represents a single row from intercom__conversation_enhanced, enriched with aggregates which determine time to first response, time to first close, and time to last close. |
+| [intercom__admin_metrics](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__admin_metrics.sql)                                               | Each record represents an individual admin (employee), enriched with admin-specific conversation data like total conversations, average rating, and median response times. |
+| [intercom__company_enhanced](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__company_enhanced.sql)                                         | Each record represents a single company, enriched with data related to the company industry, monthly spend, and user count. |
+| [intercom__company_metrics](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__company_metrics.sql)                                           | Each record represents a single row from `intercom__company_enhanced`, enriched with data like total conversation count, average satisfaction rating, median time to first response, and median time to last close with contacts associated to a single company. |
+| [intercom__contact_enhanced](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__contact_enhanced.sql)                                         | Each record represents a single contact, enriched with data like the contact's role, company, last contacted information, and email list subscription status. |
+| [intercom__conversation_enhanced](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__conversation_enhanced.sql)                               | Each record represents a single conversation, enriched with conversation part data like who was assigned to the conversation, which contact the conversation was with, the current conversation state, who closed the conversation, and the final conversation ratings from the contact. |
+| [intercom__conversation_metrics](https://github.com/fivetran/dbt_intercom/blob/master/models/intercom__conversation_metrics.sql)                                 | Each record represents a single row from `intercom__conversation_enhanced`, enriched with data like time to first response, time to first close, and time to last close. |
```

Comment:
Please double-check my edits to this table - I tried to simplify the language when possible, but I don't want to change your initial meaning.

Reply:
Initial meaning persists and changes work way better than my original. Thanks!